app/core/Tracker2/index.js

```diff
@@ -3,6 +3,7 @@ import CookieConsentTracker from './CookieConsentTracker'
 import LegacyTracker from './LegacyTracker'
 import BaseTracker from './BaseTracker'
 import GoogleAnalyticsTracker from './GoogleAnalyticsTracker'
+import DriftTracker from './DriftTracker'
```

Comment:
Curious if we need to load all these trackers throughout our application? Asking because if we need some of these only on some of the pages, it might be efficient to load those only when necessary for performance benefits.

Reply:
Thats a good question - we could definitely split these out to be imported async in the future or configure them to only initialize in situations where they are required for the user.  Something to keep in mind as we move forward!

Reply:
Yes definitely, could we consider that as part of the ongoing marketing project? I say so, because we rarely touch these things once they are set up and it also becomes riskier later and therefore might be worth considering while we are setting it up.
That said, I don't think its a blocker for shipping this if its time sensitive, but it would be worth refactoring considering the performance implications sooner rather than later. What do you think?